Luke_Smith_Portfolio_Contact.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta name="viewport" content="width=device-width initial-scale=1">
+	<title>Luke Smith Portfolio Contact</title>
+    <link rel="stylesheet" href="Luke_Smith_Portfolio.css">
+</head>
+<body>
+	<div style="background-color:white;">
+<body>
+<body>
+	<div class="topnav">
+	<div class="center">
+		<ul>
+		<li>
+			<a href="Luke_Smith_Portfolio.html">HOME</a>
+		</li>
+	  	<li>
+			<a href="Luke_Smith_Portfolio_About_Me.html">ABOUT ME</a>
+		</li>
+	  	<li>
+			<a href="Luke_Smith_Portfolio_Portfolio.html">PORTFOLIO</a>
+		</li>
+	  	<li>
+			<a href="Luke_Smith_Portfolio_Contact.html">CONTACT</a>
+		</li>
+	    </ul>
+	</div>
+	</div>
+    <div class="portfolio" style="text-align:center"> 
```

Comment:
Do not declare inline styles. Move the text-align:center to a rule in your stylesheet